docs/Makefile

```diff
@@ -20,11 +20,11 @@ help:
 
 %: Makefile
 	python make_model_tables.py -o $(BUILDDIR)/model_tables
+	rm -rf build
+	rm -rf source/_autosummary
```

Comment:
Won't this break incremental builds? Should this only occur in `make clean`?

I already have something kinda hacky here. The python script on line 22 builds some tables into `build` that I include in the docs. It needs to be defined properly with a make target (it runs even when you call `make clean`). If we really need to delete these directories then the python script needs to get called afterwards.

Reply:
woops, I meant to put this above the build tables. I'll push that now. This just ensures you get a clean build everytime you make the docs